src/shared/components/entry/__tests__/__snapshots__/SequenceView.spec.tsx.snap

```diff
@@ -393,9 +392,8 @@ exports[`SequenceView component should render SequenceInfo with provided sequenc
       </div>
     </div>
   </section>
-  <hr />
   <h3
-    id="P05067-2"
+    id="Isoform_APP770-2"
```

Comment:
Was there a particular reason to move away from the current site's isoform anchor tag format?:

```
https://www.uniprot.org/uniprot/P07954#P07954-1
```

Reply:
Because I don't have this information wherever we might link to this section. The `molecule` field is showing the name of the isoform, and not its accession.